src/shared/mosaic/mosaic_util.h

```diff
@@ -74,6 +74,8 @@ struct Node *getNextNode(struct Node *list);
 void copyNode(struct Node *node_out, struct Node node_in);
 void printNode(struct Node *list, char *str);
 int intersectInList(struct Node *list, double x, double y, double z);
+void insertIntersect(struct Node *list, double x, double y, double z, double u1, double u2, int inbound,
+                     double x2, double y2, double z2);
```

Comment:
How come we didn't need the equivalent in the `master` branch?

Reply:
I think we did, that is where @harshula grabbed it from:

https://github.com/ACCESS-NRI/MOM5/blob/master/src/shared/mosaic/mosaic_util.h#L142-L143

Reply:
Oh thanks!